packages/firebase_analytics/firebase_analytics/example/pubspec.yaml

```
@@ -6,9 +6,14 @@ dependencies:
     sdk: flutter
   firebase_analytics:
     path: ../
-  firebase_core: ^0.4.0
+  firebase_core:
+    path: ../../firebase_core/firebase_core
```

```suggestion
    path: ../../../firebase_core/firebase_core
```

Should this be like this?

---

yeah good spot 👍  I did test if by running `flutter pub get`, but it must have skipped over it and used the override path